homeassistant/components/smlight/coordinator.py

```diff
@@ -0,0 +1,106 @@
+"""DataUpdateCoordinator for Smlight."""
+
+from dataclasses import dataclass
+import logging
+import socket
+
+from pysmlight.const import Events as SmEvents
+from pysmlight.exceptions import SmlightAuthError, SmlightConnectionError
+from pysmlight.web import Api2, Info, Sensors
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.exceptions import ConfigEntryAuthFailed
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+from homeassistant.util.network import is_ip_address
+
+from .const import DOMAIN, LOGGER, SCAN_INTERVAL
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
logger is defined and imported already use that

Reply:
yeh I cleaned up inconsistencies with LOGGER